src/badguy/tarantula.cpp

```diff
@@ -41,7 +42,7 @@ static const float DROP_DETECT_RANGE = 1200.f;
 static const float RETREAT_RANGE = 4.f*32;
 
 Tarantula::Tarantula(const ReaderMapping& reader) :
-  BadGuy(reader, "images/creatures/tarantula/tarantula.sprite"),
+  BadGuy(reader, "images/creatures/tarantula/tarantula.sprite", 1),
```

Comment:
Can you please add a comment indicating what the 1 means? I saw that it appears to be the layer, but would be easier if there was a comment.